src/ProcessManager.js

```diff
@@ -33,6 +35,20 @@ class ProcessManager extends EventEmitter {
     this.emit('killed-process', pid, this.window);
   }
 
+  DebugProcess(pid) {
+      const webContentsInfo =  webContents.getAllWebContents().map(wc => ({
```

Comment:
Suggestion: I'd be more confortable passing the `webContentsId` all over rather than the `pid` to avoid this.